deploy/crds/operators_v2_catalogsourceconfig_crd.yaml

```diff
@@ -45,7 +45,11 @@ spec:
           required:
           - targetNamespace
           - packages
+          - source
           properties:
+            source:
+              type: string
+              description: The name of the OperatorSource that the operators originate from
```

Comment:
Please add validation that this is a valid resource name

Reply:
The pattern check can be made smart enough to allow that.

Reply:
The `source` is now validated with the pattern below.